eth/tracers/internal/tracetest/testdata/call_tracer/big_slow.json

```diff
@@ -47,9 +47,9 @@
     "from": "0xf8bda96b67036ee48107f2a0695ea673479dda56",
     "to": "0xb2e6a2546c45889427757171ab05b8b438525b42",
     "value": "0x0",
-    "gas": "0x22410c",
+    "gas": "0x231860",
```

Comment:
Good catch. @s1na told me about this change a while ago. You can check the PR and the actual code change [here](https://github.com/ethereum/go-ethereum/pull/27029/files#diff-6eb1a3a06566658282e72122213fdb0b756c86b312038db9ca8452063ad84417)